clippy_utils/src/consts.rs

```diff
@@ -0,0 +1,574 @@
+#![allow(clippy::float_cmp)]
```

Comment:
Weird that git doesn't recognize that this file was just moved :thinking:

Reply:
There's a small difference:
```
3c3
< use crate::utils::{clip, sext, unsext};
---
> use crate::{clip, sext, unsext};
```

Reply:
Yeah, that's fine, but git manged to figure this out for other files. That's what confuses me.

Reply:
You can tweak the git rename sensitivity ([docs](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---find-renamesltngt))